app/controllers/staff/report_forecasts_controller.rb

```diff
@@ -9,11 +9,23 @@ def show
 
     @report_presenter = ReportPresenter.new(@report)
     @report_activities = @report.reportable_activities
-    @grouped_forecasts = ForecastOverview
-      .new(@report_activities.map(&:id))
-      .latest_values
+    @total_forecast = formatted(forecasts.sum(&:value))
+    @grouped_forecasts = forecasts
+      .map { |forecast| ForecastPresenter.new(forecast) }
       .group_by { |forecast| forecast.parent_activity_id }
 
     render "staff/reports/forecasts"
   end
+
+  private
+
+  def forecasts
+    ForecastOverview
+      .new(@report_activities.map(&:id))
+      .latest_values
+  end
+
+  def formatted(bigdecimal)
+    ActionController::Base.helpers.number_to_currency(bigdecimal, unit: "£")
+  end
```

Comment:
This feels like it might belong in a view helper, rather than the controller

Reply:
Hmm, I'm not sure about this -- as at present the formatting of numbers is done in Presenters instantiated in Controllers,  rather than in Views via Helpers.

The problem is that there is no presenter for a "total" so we sum up the `Forecast#value` amounts before the forecasts are decorated by presenters and then we need to format the total. Maybe the better answer is a `TotalPresenter`?

Reply:
Oooh, yes. Very nice 👍